rplugin/python3/deoplete/sources/look.py

```diff
@@ -13,18 +14,36 @@ def __init__(self, vim):
 
         self.name = 'look'
         self.mark = '[look]'
-        self.min_pattern_length = 4
-        self.executable_look = self.vim.call('executable', 'look')
+
+        def get_neco_look_var(shortname, default):
+            return vim.vars.get('deoplete#neco_look#{}'.format(shortname), default)
+
+        self.min_pattern_length = get_neco_look_var('min_pattern_length', 4)
+        self.filetypes = get_neco_look_var('filetypes', [])
         self.is_volatile = True
 
+        self.executable_look = self.vim.call('executable', 'look')
+        self.words_source = get_neco_look_var('words_source', None)
+        if self.words_source is not None:
```

Comment:
fair enough. I like being explicit but I suppose there's no conceivable use case for an empty string.